enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -965,12 +965,11 @@ func (s *SchedulerServer) RemoveConnectedExecutor(ctx context.Context, handle *e
 	// Don't use the stream context since we want to do cleanup when stream context is cancelled.
 	ctx, cancel := context.WithTimeout(context.Background(), removeExecutorCleanupTimeout)
 	defer cancel()
-	addr := fmt.Sprintf("%s:%d", node.GetHost(), node.GetPort())
 	if err := s.deleteNode(ctx, node, nodePoolKey); err != nil {
-		log.CtxWarningf(ctx, "Scheduler: could not unregister node %q: %s", addr, err)
+		log.CtxWarningf(ctx, "Scheduler: could not unregister node %q: %s", node.GetExecutorId(), err)
 		return
 	}
-	log.CtxInfof(ctx, "Scheduler: unregistered worker node: %q", addr)
+	log.CtxInfof(ctx, "Scheduler: unregistered node %q", node.GetExecutorId())
```

Comment:
if we are logging the host in the registration message, can we also log it here for parity? 
for macs it's sometimes easier to find things by IP

Reply:
Done